cli/index.ts

```diff
@@ -126,21 +131,50 @@ createYargsCli({
         const result = await init(
           argv["project-dir"],
           {
-            warehouse: argv["warehouse"],
+            warehouse: argv.warehouse,
             gcloudProjectId: argv["gcloud-project-id"]
           },
           argv["skip-install"]
         );
 
-        console.log(commandOutputFmtString, "Directories created:");
-        result.dirsCreated.forEach(dir => console.log(dir));
-        console.log(commandOutputFmtString, "Files written:");
-        result.filesWritten.forEach(file => console.log(file));
+        writeStdOut(commandOutput("Directories created:"));
+        result.dirsCreated.forEach(dir => writeStdOut(dir));
+        writeStdOut(commandOutput("Files written:"));
+        result.filesWritten.forEach(file => writeStdOut(file));
         if (result.installedNpmPackages) {
-          console.log(commandOutputFmtString, "NPM packages successfully installed.");
+          writeStdOut(commandOutput("NPM packages successfully installed."));
         }
       }
     },
+    {
+      format: "init-creds <warehouse> [project-dir]",
```

Comment:
Why do we need to re-specify the warehouse at all if we can get it from the project dir? Seems like we could easily avoid having the user specify the warehouse twice here.

It looks like the project-dir is required, in which case i think it should be `<project-dir>` in the format string.

Reply:
So, `project-dir` is not actually required, it defaults to the current directory if unset.

I suppose we could make `warehouse` optional too, and if it's unset, try to look in `project-dir/dataform.json`, would you prefer that?

Reply:
Ahh ok, that argument is really more of an output folder for the file. Makes sense! I thought you were reading properties from the project but you aren't, I think I prefer the explicit approach for now, unless we decide to tie this command more strongly to a project dir.